service/integration/attribute_fqns_test.go

```diff
@@ -244,8 +244,14 @@ func (s *AttributeFqnSuite) TestGetAttributeByFqn_WithKeyAccessGrants_Definition
 	localKAS, err := s.db.PolicyClient.CreateKeyAccessServer(s.ctx, &kasregistry.CreateKeyAccessServerRequest{
 		Uri: "https://example.org/kas2",
 		PublicKey: &policy.PublicKey{
```

Comment:
This looks great, @suchak1. Can you please add assertions to 1-2 tests in this suite and those in `kas_registry_test` that the key found in the KAS or Grant was the same as the 'cached' key used to create?

It looks like we had assertions around those for the local keys previously but none covering assertion that a KAS created with a cached key finds the same cached key on lookup (which tests our unmarshaling/marshaling of the proto to and from the db is sound).

Reply:
added assertions to both test files